src/hotspot/share/cds/dumpTimeClassInfo.hpp

```diff
@@ -42,7 +42,8 @@ class DumpTimeClassInfo: public CHeapObj<mtClass> {
   bool                         _excluded;
   bool                         _is_early_klass;
   bool                         _has_checked_exclusion;
-
+  bool                         _is_required;
```

Comment:
Is this flag only used for hidden classes?

Reply:
I changed it to `_is_required_hidden_class`.